src/main/java/uk/gov/hmcts/reform/userprofileapi/controller/UserProfileController.java

```diff
@@ -67,42 +71,43 @@
 
 
     @ApiOperation(value = "Create a User Profile",
-                  authorizations = {
+            authorizations = {
                     @Authorization(value = "ServiceAuthorization"),
-                    @Authorization(value = "Authorization") 
-                  })
+                    @Authorization(value = "Authorization")
+            })
     @ApiResponses({
-        @ApiResponse(
-            code = 201,
-            message = "Create a User Profile using request body",
-            response = UserProfileCreationResponse.class
-        ),
-        @ApiResponse(
-            code = 400,
-            message = "Bad Request",
-            response = String.class
-        ),
-        @ApiResponse(
-            code = 409,
-            message = "User already exists",
-            response = String.class
-        ),
-        @ApiResponse(
-                code = 429,
-                message = "Too many request for re invite",
-                response = String.class
-        ),
-        @ApiResponse(
-            code = 500,
-            message = "Internal Server Error",
-            response = String.class
-        )
+            @ApiResponse(
+                    code = 201,
+                    message = "User Profile created successfully",
+                    response = UserProfileCreationResponse.class
+            ),
+            @ApiResponse(
+                    code = 400,
+                    message = "An invalid request has been provided"
+            ),
+            @ApiResponse(
+                    code = 403,
+                    message = FORBIDDEN_ERROR_ACCESS_DENIED
```

Comment:
It is good to have constants but here either keep all messages in constants or keep the string for 403. It will look uniform.

Reply:
Will revert the 403 Constant to keep uniform look

Reply:
This has been addressed in the following commit: https://github.com/hmcts/rd-user-profile-api/pull/302/commits/3ed359799ee8b41134c479fd74c9aba7316f5626